docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayFixed.xml

```diff
@@ -183,6 +186,17 @@
           FALSE.</entry>
         </row>
 
+        <row>
+          <entry><emphasis>noCommon</emphasis></entry>
+
+          <entry>Optional. A boolean TRUE or FALSE flag for "commoning up" of
```

Comment:
This doesn't really tell you what the flag does.  E.g. why would you set it or not.  "It can have a significant impact on performance" - but doesn't explain how or why.